1.Exploratory_Data_Analysis/scripts/2.extended_umap_analysis.ipynb

```diff
@@ -0,0 +1,612 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    data = data[data.oneb_Metadata_Treatment_Dose_Inhibitor_Dose.isin(treatments)]
</pre><p>This is just a personal preference, but I like to access columns using brackets, instead of like a function for readability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>